### ISSUE TITLE
open position

### ISSUE THREAD

**Issue by Pandena** [papstuc/counterstrike2#5]
Hello, are you still an active developer in cs? We would like to hire you to work on a p2p skin trading system for our platform

**Comment by papstuc** [comment]
Yes, I still actively develop stuff for this game. You can add me on Discord (bedaisy.sys is my username) if you want to.
